crates/compiler/parse/src/ast.rs

```diff
@@ -550,7 +553,7 @@ pub fn is_expr_suffixed(expr: &Expr) -> bool {
         // expression without arguments, `read!`
         Expr::Var { .. } => false,
 
-        Expr::TaskAwaitBang(..) => true,
+        Expr::TaskAwaitBang(..) | Expr::ResultTryQuestion(..) => true,
```

Comment:
Let's also update `is_top_level_suffixed` function in this file. The behaviour should be the same as for `TaskAwaitBang`

Reply:
After that, some of snapshots will change. In particular, it will affect the postfix in the statement position